testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py

```diff
@@ -0,0 +1,219 @@
+"""
```

Comment:
Everything is commented. Remove it, if unnecessary.

Reply:
There are encryption tests, we should not remove it as the plan is to get back the feature in 2.1.0. It supposed to be commented. Anyways, now with 2.1.0, feature is already back, I uncommented the code now